test/test_judiWeights.jl

```diff
@@ -91,9 +91,11 @@ ftol = 1f-6
     @test isequal(length(w_vcat), 2*length(w))
     @test isequal(w_vcat.nsrc, 2*w.nsrc)
 
-    # dot, norm, abs
+    # dot, norm, abs, max, min
     @test isapprox(norm(w), sqrt(dot(w, w)))
-    @test isapprox(abs.(w.weights[1]), abs(w).weights[1]) 
+    @test isapprox(abs.(w.weights[1]), abs(w).weights[1])
+    @test isapprox(maximum(w),max([maximum(w.weights[i]) for i=1:w.nsrc]...))
+    @test isapprox(minimum(w),max([minimum(w.weights[i]) for i=1:w.nsrc]...))
```

Comment:
That's the idea but would make it even more explicit. Currently this test "the right hand side does the same as copy pasting what's in the code", which is ok, but what you actually want is "I get exactly the value I am supposed to get". For example below where I choose what's in `d_ones` then explicitly check what the norm value is.


Like put the maximum value in `w`
```
w.weights[1][1,1] = 1f10
@test maximum(w) == 1f10
```

sand same for min.